### ISSUE TITLE
Pointer to an optional variable-sized byte array

### ISSUE THREAD

**Issue by blueshade7** [harfbuzz/harfbuzz#1417]
In an accelerator, I need to store a pointer to an optional table represented as a variable-sized byte array with its size stored externally. I built it based on `UnsizedArrayOf<HBUINT8>` and used `&Null()` to indicate a missing table. This implementation (`Dict` in cff-subset branch) no longer compiles after the fix for issue #1300. What is the right approach in this situation?

**Comment by blueshade7** [comment]
I can use `(const Dict*)0` instead, but no safer than `&Null(Dict)`

**Comment by blueshade7** [comment]
Another possible workaround is to add `DEFINE_SIZE_MIN (0)` to my struct so that `null_size` is defined and `Null()` may be created. Now inclining to this.

**Comment by behdad** [comment]
Yes.  Do that.  See hb-ot-glyf-table.hh  Add a comment why it's safe (that you won't access it.)
